assets/common/common.go

```diff
@@ -100,6 +104,32 @@ func ValidateAndBuildPostBodyPostMessage(input ConcourseInput) (url.Values, erro
 	return data, nil
 }
 
+func ValidatePostMessageAttachments(attachments string, attachmentsFile string) (string, error) {
+  if attachments != "" && attachmentsFile != "" {
+    err := errors.New(fmt.Sprintf("cannot supply both attachmentsFile and attachments for Chat.PostMessage"))
+    return "", err
+  }
+  if attachments == "" && attachmentsFile == "" {
+    err := errors.New(fmt.Sprintf("must supply one of attachmentsFile or attachments for Chat.PostMessage"))
+    return "", err
+  }
+
+  if attachments != "" {
+    return attachments, nil
+  } else {
+    attachmentsStringRawBytes, err := ioutil.ReadFile(attachmentsFile)
+    if err != nil {
+      return "", err
+    }
+    attachmentsStringRaw := string(attachmentsStringRawBytes)
+    if attachmentsStringRaw == "" {
+      err := errors.New(fmt.Sprintf("attachments file is empty"))
+      return "", err
+    }
+    return attachmentsStringRaw, nil
```

Comment:
Should we validate for correct JSON format here before returning?

Reply:
oooh, I do like that.  I'll add a jira for this instead of doing it here. That cool?

Reply:
cool